data/registry/instrumentation-dotnet-azure-sdk.yml

```diff
@@ -15,3 +15,4 @@ authors:
 urls:
   repo: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md#opentelemetry-with-azure-monitor-zipkin-and-others
 createdAt: 2021-12-16
+isFirstParty: true
```

Comment:
@svrnm This is from Azure, not open-telemetry, so I marked `isFirstParty` to `true`.

Reply:
yes, here it makes sense to set this as isFirstParty